src/ray/protobuf/usage.proto

```diff
@@ -66,6 +66,12 @@ enum TagKey {
   SERVE_USER_CONFIG_LIGHTWEIGHT_UPDATED = 16;
   // Whether autoscaling_config changed as a lightweight config update
   SERVE_AUTOSCALING_CONFIG_LIGHTWEIGHT_UPDATED = 17;
+  // Whether the serve.status API was used
+  SERVE_STATUS_API_USED = 22;
```

Comment:
I know there could be some merge conflicts with other PRs if you start with 18. But I feel we shouldn't skip numbers until then. Whichever PR merged later should resolve the conflicts. Also, would be nice if we can add comments like `("1" if used)`

Reply:
Good point, I've changed these to start at 18!